.travis/Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get -qq update \
   && rosdep install -y --from-paths src --ignore-src \
   && apt-get clean \
   && rm -rf /var/lib/apt/lists/* \
-  && sh -c 'curl -sL https://raw.githubusercontent.com/at-wat/gh-pr-comment/master/install.sh | sh -s'
+  && sh -c 'curl -sL https://raw.githubusercontent.com/at-wat/gh-pr-comment/master/install.sh | sh -s latest /catkin_ws'
```

Comment:
Fetching GitHub Release files often fails without github token on CI environment due to API rate limit.
See https://github.com/at-wat/mcl_3dl/pull/329/files

Reply:
thank you!